src/html/candidate-detail.html

```diff
@@ -0,0 +1,30 @@
+<html>
+  <head>
+    <title>{{title}}</title>
```

Comment:
Might have missed something, but I don't see where this actually gets rendered

Reply:
You're correct, this was left over from an earlier attempted approach and should be taken out.